waveform_editor/gui/main.py

```diff
@@ -98,7 +98,7 @@ def __init__(self):
         self.template = pn.template.FastListTemplate(
             title=f"Waveform Editor (v{waveform_editor.__version__})",
             main=self.tabs,
```

Comment:
Should main be a list as well? See the example listed in the docs: https://panel.holoviz.org/reference/templates/FastListTemplate.html#tips-tricks